src/catchup/ApplyBucketsWork.cpp

```diff
@@ -31,12 +31,12 @@ ApplyBucketsWork::ApplyBucketsWork(
     , mFirstVerified(firstVerified)
     , mApplying(false)
     , mLevel(BucketList::kNumLevels - 1)
-    , mBucketApplyStart(app.getMetrics().NewMeter(
```

Comment:
from what I see you replaced meters by counters, which seems to be more of a downgrade as a meter tracks both count and rate, where as a counter only tracks count.
If you were looking at getting access to the counter stored inside a meter, see [`medida::Meter::count`](http://dln.github.io/medida/api/classmedida_1_1Meter.html#afb53f91f1ef6a53e33c82d501353443b)

Reply:
I've changed that so I can reset (counters can be reset, I don't see that option for meters). I need resets for tests.

Reply:
Oh I see - it seems better to just add a `clear` method to `Meter`. It seems like a legitimate change to the library that can also be submitted upstream if you feel like it (but for now you can just make the change in our fork as that project seems to be abandoned at the moment).